relays/bin-substrate/src/rialto_millau/mod.rs

```diff
@@ -57,10 +54,10 @@ async fn run_relay_messages(command: cli::RelayMessages) -> Result<(), String> {
 			type Source = Millau;
 			type Target = Rialto;
 
-			let source_client = source_chain_client::<Source>(source).await?;
-			let source_sign = Source::source_signing_params(source_sign)?;
-			let target_client = target_chain_client::<Target>(target).await?;
-			let target_sign = Target::target_signing_params(target_sign)?;
+			let source_client = source.into_client::<Source>().await.map_err(format_err)?;
```

Comment:
Could we not change `run_relay_messages` to return an `anyhow::Result` here instead?

Reply:
Yes, but `run_relay_messages` is removed in #858 anyway, so I didn't want to cause more conflicts.